specification/resources/resource-manager/Microsoft.Resources/preview/2024-06-01-preview/examples/ListChanges.json

```diff
@@ -0,0 +1,72 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "myResourceGroup",
+    "resourceProviderNamespace": "Microsoft.Storage",
+    "resourceType": "storageAccounts",
+    "resourceName": "myStorageAccount",
+    "api-version": "2024-06-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.Storage/storageAccounts/myStorageAccount/providers/Microsoft.Resources/changes/a9f34285-13a2-e79c-f468-cfb71c7bd227",
+            "name": "a9f34285-13a2-e79c-f468-cfb71c7bd227",
+            "type": "Microsoft.Resources/changes",
+            "properties": {
+              "targetResourceId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.Storage/storageAccounts/myStorageAccount",
+              "targetResourceType": "Microsoft.Storage/storageAccounts",
+              "changeType": "Update",
+              "changeAttributes": {
+                "correlationId": "88420d5d-8d0e-471f-9115-10d34750c617",
+                "timestamp": "2021-11-19T14:29:09.9210000Z",
+                "changesCount": 2,
+                "previousResourceSnapshotId": "ed90e35a-1661-42cc-a44c-e27f508005be",
+                "newResourceSnapshotId": "6eac9d0f-63b4-4e7f-97a5-740c73757efb",
+                "changedBy": "9500bb33-43da-4588-80b8-f65430dd937a",
+                "changedByType": "Application",
+                "clientType": "SDK",
+                "operation": "Microsoft.Namespace/ResourceType/write",
+                "changeControlId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/myResourceGroup/providers/Microsoft.AzureChangeControl/changecontrols/myChangeControl"
+              },
+              "changes": {
+                "properties.provisioningState": {
+                  "previousValue": "Updating",
+                  "newValue": "Succeeded"
+                },
+                "tags.key1": {
+                  "previousValue": "null",
```

Comment:
Would be great if your examples covered all the types that JSON can represent.

Reply:
Added example for an array elements value being changed